src/test/schema_test.cpp

```diff
@@ -1430,26 +1430,29 @@ TEST(SchemaTest, DemultiplexerConfigDemultiplyCountNegativeNotAllowed) {
     EXPECT_EQ(result, ovms::StatusCode::JSON_INVALID);
 }
 
-TEST(SchemaTest, DemultiplexerConfigDemultiplyCountEqualsZeroNotAllowed) {
+TEST(SchemaTest, DemultiplexerConfigDemultiplyCountEqualsZeroAllowed) {
+    // this is to allow dynamic demultiplexing
     const std::string demultiplyCountToReplace{"\"demultiply_count\": 10"};
     const std::string demultiplyCount{"\"demultiply_count\": 0"};
     std::string config(demultiplexerConfig);
     config.replace(config.find(demultiplyCountToReplace), demultiplyCountToReplace.size(), demultiplyCount);
     rapidjson::Document demultiplexerConfigDemultiplyCountEqualsZeroParsed;
     demultiplexerConfigDemultiplyCountEqualsZeroParsed.Parse(config.c_str());
     auto result = ovms::validateJsonAgainstSchema(demultiplexerConfigDemultiplyCountEqualsZeroParsed, ovms::MODELS_CONFIG_SCHEMA);
-    EXPECT_EQ(result, ovms::StatusCode::JSON_INVALID);
+    EXPECT_EQ(result, ovms::StatusCode::OK);
 }
 
-TEST(SchemaTest, DemultiplexerConfigDemultiplyCountEqualsOneNotAllowed) {
+TEST(SchemaTest, DemultiplexerConfigDemultiplyCountEqualsOneAllowed) {
+    // this is to allow dynamic demultiplexing. We could invalidate 1 and still accept 0, [2,3, ...]
+    // but this is just probably not efficient - unnecessary to have demultiply_count == 1
```

Comment:
There could be use case for 1. For example when library has dynamic batch but administrator wants to enforce batch to equal 1 and make other batch fail pipeline execution.